### PR TITLE
Mint Sign-Up Page

### PR DESCRIPTION
**Pull request by KinezaEnaelle**
This is a data collection App designed by Uchenna Martins Anya and Iris Enaelle Kineza, using HTML5 and CSS3. We implemented linter to help us easily format our  code  and we made use of prettier, weblint and stylelint

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required 

## :clap::clap::clap::clap: Amazin work!! The website looks pretty similar to the original one. 

Things to notice:
:100: Using semantic HTML for main sections and elements of the form.
:100: README descriptive enough
:100: Link to Live Preview! this made my life more easier! thank you :smiley: !!!

## Changes Required

### [Mandatory]
- [ ] Project not passing all linter's checks. 

- [ ] Remove unnecessary files form project. Only upload the ones related to the project.
image

- [ ] Remove all inline styling from HTML and assign it as a selector in the CSS file. Is best practice to use a stylesheet for all styling purposes. This helps also debugging the CSS if something is not displaying correctly. 
image


### [OPTIONAL]

- [ ] Improve the link name to the live preview. Something similar to this example:

[Mint sign-up page clone](https://rawcdn.githack.com/KinezaEnaelle/Mint-Signup/a945d8e625863ffe14e6a76fa53725a5b92dc06a/index.html)

- [ ] Try removing the ```<div>``` tag that wraps the ```<label>``` html tag.

- [ ] Prefer CSS classes over general selectors. As it is more specific. Example:

:-1:  From this pattern
```
header button {
    ... properties
}
```
:+1: to this
```
.header_btn {
    ... properties ...
}
```

- [ ] For future projects try to make Commit Messages more descriptive. This will help the reader understand the decisions and changes made in the past, and grasp a better understanding of what the programmer wanted to do. Example:

:-1:  From this. This makes the reader wonder what was the change referring to.
```
Readme
```
:+1: to This. Still short but the reader and you can follow the changes made.
```
Update: add contact info to README
```

## Keep up the great work :muscle:, and Happy coding :smiley: :computer: !!!

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Good job fixing the linters, team! 👍 
Very good use of semantic HTML, and your code is clean 💯 
But we still some required changes to implement.
As the previous TSE pointed before, kindly make the following changes:

- Avoid inline styling, it is a better practice to add CSS classes.
- Remove unnecessary files form project. Add them to `.gitignore`
- Use a form tag to wrap your form. It is the way you can submit and manage the data the user inputs.
   - It uses a `<form>` tag ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))


[OPTIONAL]
- It is a better practice to add some CSS classes and not to use general selectors to style the content of your page.

# Keep it up!! 💪

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+- Linkedin: [Iris Enaelle Kineza](https://www.linkedin.com/in/iris-enaelle-kineza-25a676187/)
+
+## Contributions
+
+Faeture requests and issues are most welcome.
+
+## Live Demo 
+
+https://rawcdn.githack.com/KinezaEnaelle/Mint-Signup/a945d8e625863ffe14e6a76fa53725a5b92dc06a/index.html
+
```
Comment (arslanbisharat) [inline README.md]: your link should be wrapped in one to two words

File: index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+            <p class="privacy">
+              <em>By clicking Create account, you agree to our</em>
+              <a href="https://www.mint.com/terms" target="blank">Terms</a>
+              <em>have read and acknowledge our </em>
+              <a href="https://www.intuit.com/privacy/" target="blank"
+                >Privacy Statement</a
+              >
+            </p>
+          </section>
```
Comment (arslanbisharat) [inline index.html]: Please use **form** tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            <img src="assets/images/quickbooks-logo.PNG" alt="logo" />
+          </picture>
+          <picture>
+            <img
+              src="assets/images/logo-turbotax.PNG"
+              alt="logo"
+              class="turbo"
+            />
+          </picture>
+        </div>
```
Comment (arslanbisharat) [inline index.html]: It shows the Mint logo and text (logo on the left, text on the right) 

 It shows the Quickbooks logo and text (logo on the left, text on the right

 It shows the Turbotax logo and text (logo on the left, text on the right)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x] Your PR is not descriptive. Please add more about your project in the PR message. This is a place where you can tell TSEs what you di din this project what you are proud of and what you additionally added.

- [x] mint, QuickBooks, and TurboTax icons should have a text with them. You cannot add only images there. 
chrome-capture (1)

- [x] You should use form tag to warp form elements like inputs, labels, etc as suggested by previous TSE.

- [x] **Optional** please try to add the link into one or two works like [Live demo](https://rawcdn.githack.com/KinezaEnaelle/Mint-Signup/a945d8e625863ffe14e6a76fa53725a5b92dc06a/index.html)
chrome-capture (2)

- [x] **Optional** About section is not descriptive. Please tell within 1-2 lines what you did in this project and what technologies you used, etc 


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by javitocor — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED ♻️ 

Great job implementing all the changes! 👍 
The linters still green, well done! 💚 
I can see you have been working hard in this project 💪 
Just one more thing to get the project approved and move forward ⏩ 
Let's make all the projects to be as professional as we can, we want to have good repositories when a professional recruiter comes to take a look at it,  we want them to get impressed.
## Changes
- Your live link is not updated, I cannot see the form tag in it, you made the changes in your repo, and we want that anyone who checks the live link and inspect it, will be able to see that you can use semantic HTML tags. Kindly update it

Once you have implemented the change, please resubmit for a new code review and move forward into the next project.

Good luck 😃

**Review by javitocor — APPROVED** [review]
# STATUS APPROVED 🌟 🥇 

Really good job guys 👍 
You have met all the requirements! 💯 

Remember to submit the completion form and you can move forward into the next project ⏩ 

Good luck 😃

**Pull request merged**